Rephrase linear gradient explanation

steviehailey opened the pull request:
Small tweak to explanation to remove ambiguity over horizontal vs vertical

Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #39989

<!-- Feel free to add any additional description of changes below this line -->

steviehailey committed:
Rephrase linear gradient explanation

Small tweak to explanation to remove ambiguity over horizontal vs vertical

RandellDawson commented on curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-gradual-css-linear-gradient.md:
```diff
@@ -10,7 +10,7 @@ forumTopicId: 301047
 Applying a color on HTML elements is not limited to one flat hue. CSS provides the ability to use color transitions, otherwise known as gradients, on elements. This is accessed through the <code>background</code> property's <code>linear-gradient()</code> function. Here is the general syntax:
 <code>background: linear-gradient(gradient_direction, color 1, color 2, color 3, ...);</code>
-The first argument specifies the direction from which color transition starts - it can be stated as a degree, where 90deg makes a vertical gradient and 45deg is angled like a backslash. The following arguments specify the order of colors used in the gradient.
+The first argument specifies the direction from which color transition starts - it can be stated as a degree, where 90deg makes a gradient from left to right and 45deg is angled like a backslash. The following arguments specify the order of colors used in the gradient.
```
@steviehailey You have made a great catch here!  I would make one small adjustment to clarify that `90deg` creates a `horizontal` gradient.  For example:
```suggestion
The first argument specifies the direction from which color transition starts - it can be stated as a degree, where `90deg` makes a horizontal gradient (from left to right) and `45deg` is angled like a backslash. The following arguments specify the order of colors used in the gradient.
```
If you agree with this suggestion, you can simply click the `Commit suggestion` button and add an applicable commit message for a second commit.

erictleung replied:
@RandellDawson should we be using backticks for code, or wrapping them around `<code>` tags? I was going to suggest `<code>` tags, but maybe something has changed and we need to [update the contributor docs](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=formatting-challenge-text).

steviehailey replied:
Hey @erictleung I'd agree those docs do suggest I should swap the back tick for code tags
> References to code parts (i.e. function, method or variable names) should be wrapped in <code> tags
Will push another commit...

Sky020 replied:
@erictleung Just to mention it: The docs do say
> Note: The equivalent Markdown should be used, where applicable, in place of HTML tags.

erictleung replied:
@Sky020 ah, thanks! I didn't read down the page far enough 😅 Maybe we can reword the docs to make it less confusing then, but that's another issue beyond the scope of this PR

RandellDawson reviewed (changes requested):
@steviehailey See my suggestion.

erictleung commented on curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-gradual-css-linear-gradient.md:
```diff
@@ -10,7 +10,7 @@ forumTopicId: 301047
 Applying a color on HTML elements is not limited to one flat hue. CSS provides the ability to use color transitions, otherwise known as gradients, on elements. This is accessed through the <code>background</code> property's <code>linear-gradient()</code> function. Here is the general syntax:
 <code>background: linear-gradient(gradient_direction, color 1, color 2, color 3, ...);</code>
-The first argument specifies the direction from which color transition starts - it can be stated as a degree, where 90deg makes a vertical gradient and 45deg is angled like a backslash. The following arguments specify the order of colors used in the gradient.
+The first argument specifies the direction from which color transition starts - it can be stated as a degree, where 90deg makes a gradient from left to right and 45deg is angled like a backslash. The following arguments specify the order of colors used in the gradient.
```
While we're already making changes, we might as well tidy this small bit up as well.

```suggestion
The first argument specifies the direction from which color transition starts - it can be stated as a degree, where <code>90deg</code> makes a gradient from left to right and <code>45deg</code> is angled like a backslash. The following arguments specify the order of colors used in the gradient.
```

erictleung reviewed (commented):
Great change! I think you're right that it is a horizontal. This explanation make sense.

I've made some minor suggestions that shouldn't be too significant but would take care of some formatting while we're here. I hope you don't mind 😄 

**Edit**: I guess @RandellDawson beat me to this review before I was able to refresh my page 😅 please refer to his suggestions.

steviehailey committed:
Update curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-gradual-css-linear-gradient.md

Code review suggestion - mention horizontal and quote the values

Co-authored-by: Randell Dawson <5313213+RandellDawson@users.noreply.github.com>

RandellDawson commented:
LGTM

steviehailey committed:
Use code tag for snippets

Was using backtick but updated based on formatting guidelines - https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=formatting-challenge-text

steviehailey commented:
> LGTM

Thanks @RandellDawson - 1 additional change pushed to use code tags in favour of backticks (so lost your approval)

majestic-owl448 reviewed (approved):
LGTM :+1: 🥳

erictleung commented:
@steviehailey sorry for the confusion with the code formatting! I would go with whatever @Sky020 and @RandellDawson suggest with either using <code>``</code> or `<code></code>` tags. I wasn't aware they were interchangeable with our challenge parser.

RandellDawson commented:
@erictleung In the very near future, all `<code>` sections will be transformed to single backticks, so it does not matter.  Both work currently in the `description` and `instructions` sections.

RandellDawson commented:
Congrats @steviehailey on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝
